packages/blockchain-wallet-v4-frontend/src/modals/Onboarding/Welcome/index.tsx

```diff
@@ -104,26 +113,25 @@ class WelcomeContainer extends React.PureComponent<Props> {
               />
             </Text>
           </Header>
-          <HeaderDesc weight={500}>
-            <FormattedMessage
-              id='modals.wallet.welcome.desc'
-              defaultMessage='Let’s get your wallet set up. You can quickly buy crypto now or do this later.'
-            />
-          </HeaderDesc>
-          <Card>
-            <Icon color='blue600' name='cart-filled' size='32px' />
-            <Title lineHeight='30px' size='20px' weight={600}>
+          <ContentContainer>
+            <IconBackground>
+              <Icon color='blue600' name='cart-filled' size='25px' />
+            </IconBackground>
+            <Title size='20px' weight={600}>
               <FormattedMessage
                 id='modals.wallet.welcome.sb.title'
-                defaultMessage='Buy Crypto'
+                defaultMessage='Own Crypto in a Few Minutes'
               />
             </Title>
-            <Content lineHeight='24px' weight={500}>
+            <ContentTextWrapper color='grey600' size='14x' weight={500}>
               <FormattedMessage
-                id='modals.wallet.welcome.sb.content'
-                defaultMessage='We can help you buy in just a few simple steps'
+                id='modals.wallet.welcome.sb.verify'
+                defaultMessage='Verify your identity and buy crypto with a bank transfer '
               />
-            </Content>
+            </ContentTextWrapper>
+            <Image name='buy-wallet' width='282px' height='165px' />
```

Comment:
Width fine as long as it looks good on mobile, otherwise make a percentage please

Reply:
Thanks for the reminder. Looks good on everything, a bit big on iPhone5 but fits with enough room around.